src/assets/icons/AccountIcon.tsx

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
+import {ViewStyle} from 'react-native';
 import Svg, {G, Path} from 'react-native-svg';
+interface Props {
+  style: ViewStyle | ViewStyle[];
+}
 
-const AccountIcon = () => {
+const AccountIcon = (props: Props) => {
   return (
-    <Svg width="26" height="24" viewBox="0 0 26 24">
+    <Svg width="26" height="24" viewBox="0 0 26 24" style={props.style}>
       <G fill="none" fill-rule="evenodd">
         <G fill-rule="nonzero">
```

Comment:
please note the SVG attributes in react becomes camelCase
i.e attributes like stroke-width and fill-rule become strokeWidth and fillRule, etc.